www-client/elinks/elinks-9999.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 PYTHON_COMPAT=( python2_7 )
```

Comment:
Any chance of something newer than Python 2? There's a real push to get rid of Python 2 where possible.